tjpcov/covariance_cluster_counts.py

```diff
@@ -0,0 +1,117 @@
+from .covariance_clusters import CovarianceClusters
+import numpy as np
+import pyccl as ccl
+from scipy.integrate import romb
+
+
+class ClusterCounts(CovarianceClusters):
+    """Implementation of cluster covariance that calculates the autocorrelation
+    of cluster counts (NxN)
+    """
+
+    cov_type = "fourier"
```

Comment:
This should be one of `gauss`, `SSC` or `NG`. In this case it would be `gauss` (and SSC for the class that implements the covariance for the clusters x Cell)

Reply:
I made it gauss for now but we should separate into two classes - I can do in a future PR.